crates/ruff_linter/resources/test/fixtures/flake8_type_checking/quote3.py

```diff
@@ -53,3 +53,17 @@ def f():
     def test_annotated_non_typing_reference(user: Annotated[str, Depends(get_foo)]):
         pass
 
+
+def f():
+    from typing import Literal
+    from third_party import Type
+
+    def test_string_contains_opposite_quote_do_not_fix(self, type1: Type[Literal["'"]], type2: Type[Literal["\'"]]):
+        pass
+
+def f():
```

Comment:
nit: we don't enforce formatting on the test fixtures but it's still useful to have consistency
```suggestion
    def test_string_contains_opposite_quote_do_not_fix(self, type1: Type[Literal["'"]], type2: Type[Literal["\'"]]):
        pass


def f():
```